### DISCUSSION TITLE
LoaderFunction Called Twice with Weird Params in Joke Tutorial

### DISCUSSION THREAD

**Discussion by jamesqquick** [remix-run/remix#3205] [Q&A] [answered]
I'm following the [Jokes Tutorial](https://remix.run/docs/en/v1/tutorials/jokes) and have come across an interesting issue. On the dynamic route to display a joke by id (`/jokes/$jokeId`), my loader function seems to be getting called twice.

The first time it gets called, the `jokeId` appears to be correct with the value of the joke id. BUT the second time it's called, `Roboto-Regular.ttf` is listed as the `jokeId` route parameter. I'm assuming the application is trying to load the Roboto font for some reason at `/jokes/Roboto-Regular.ttf` but I'm not sure why that is happening?

![CleanShot 2022-05-16 at 11 04 00](https://user-images.githubusercontent.com/5391915/168635847-ad969aa5-51e9-48ce-b0ab-a22497f6cfeb.png)

**Comment by kiliman** [comment]
Did you change your CSS to include this font? If so, looks like you're using a relative URL instead of an absolute one.

**Comment by jamesqquick** [comment]
@kiliman that's a great suggestion and one that I was looking into as well. I can't find any reference in my code to a relative URL in that format which is why I'm confused. I basically copied all of the CSS, but the only import I see is for the Baloo font which is local. I also only see Roboto listed in one place. 
![CleanShot 2022-05-16 at 11 57 35](https://user-images.githubusercontent.com/5391915/168644793-e0b2561f-9f62-4c10-92fd-6520d9c76a8b.png)

**Reply by kiliman** [reply]
Hmm.. that's really odd then. What does it look like in your Network tab?

Find where it tries to load that font, then see what the Initiator is.

<img width="913" alt="image" src="https://user-images.githubusercontent.com/47168/168645473-3366df9c-f5b6-4dd2-8ac9-186e42af7481.png">

**Comment by jamesqquick** [comment]
Looks like something in the jokes route. Double checked the CSS imports there and it seems to be pulling the correct project CSS file. I'm really not sure what to do look at lol 
![CleanShot 2022-05-16 at 12 45 58](https://user-images.githubusercontent.com/5391915/168652270-4e93a497-e79b-4a0d-aaba-1f5d26f4b2b8.png)

**Reply by kiliman** [reply]
That's really weird. What are all those other fonts? I don't see any of those fonts in the tutorial. Have you tried clearing out your build and public/build folders and restarting the app?

**Comment by jamesqquick** [comment, marked as answer]
Just deleted cache and build folders and restarted, but still there. As far as the other fonts, I had no idea, BUT I just figured it out. It's from an extension called BlackBox which turns out was giving me issues in other places as well :( So, this had NOTHING to do with Remix, but thank you so much for helping debug!!

**Reply by jamesqquick** [reply]
Feel free to close :)
